evcache-core/src/main/java/com/netflix/evcache/pool/EVCacheClient.java

```diff
@@ -153,6 +155,7 @@
         this.shouldEncodeHashKey = EVCacheConfig.getInstance().getPropertyRepository().get(this.serverGroup.getName() + ".hash.encode", Boolean.class).orElse(null);
         this.maxDigestBytes = EVCacheConfig.getInstance().getPropertyRepository().get(this.serverGroup.getName() + ".max.digest.bytes", Integer.class).orElse(null);
         this.maxHashLength = EVCacheConfig.getInstance().getPropertyRepository().get(this.serverGroup.getName() + ".max.hash.length", Integer.class).orElse(null);
+        this.encoderBase = EVCacheConfig.getInstance().getPropertyRepository().get(this.serverGroup.getName() + ".hash.encoder", String.class).orElse("base64");
```

Comment:
Maybe we should also include an app level encoder in EVCacheImpl, that will then be used when instantiating EVCacheKey. This will be helpful when we want to use something other than base64 for all the ASGs, without need to add an override for each ASG.